modules/govuk/manifests/node/s_cache.pp

```diff
@@ -45,13 +45,29 @@
   include router::gor
   include nscd
 
+  # The next three blocks increase file descriptor and IP connection
+  # tracking limits on cache machines so the router can handle more
+  # simultaneous connections. The router is the only app on these
+  # machines, so we can afford for it to use more resources.
   limits::limits { 'deploy_nofile_router':
     ensure     => present,
     user       => 'deploy',
     limit_type => 'nofile',
     both       => 65535,
   }
 
+  govuk_harden::sysctl::conf { 'cache-nf-conntrack-limit':
+    content => "net.netfilter.nf_conntrack_count = 131072\n",
```

Comment:
I think `nf_conntrack_count` is the current count of the number of connections, whereas we should be setting the `nf_conntrack_max` variable here to increase the limit. I also wonder if we should look at increasing the `nf_conntrack_buckets` variable instead because the docs say that `nf_conntrack_max` will be set to 4 times that value.

https://www.kernel.org/doc/Documentation/networking/nf_conntrack-sysctl.txt

Reply:
Nice one, although it's weird that the cache machines have 16GB RAM but only 16384 buckets...

Reply:
Yeah, I noticed that too and thought it was weird... 🤔

Reply:
I've manually upped the limit